be/src/vec/functions/function_utility.cpp

```diff
@@ -70,13 +70,23 @@ class FunctionSleep : public IFunction {
 
     bool use_default_implementation_for_nulls() const override { return false; }
 
+    bool use_default_implementation_for_constants() const override { return false; }
+
     Status execute_impl(FunctionContext* context, Block& block, const ColumnNumbers& arguments,
                         size_t result, size_t input_rows_count) const override {
         ColumnPtr& argument_column = block.get_by_position(arguments[0]).column;
 
         auto res_column = ColumnUInt8::create();
 
-        if (auto* nullable_column = check_and_get_column<ColumnNullable>(*argument_column)) {
+        if (is_column_const(*argument_column)) {
```

Comment:
why change this code?

Reply:
have set use_default_implementation_for_constants = false, so need deal with const column